### ISSUE TITLE
Accessibility - Lighthouse: Image Optimization

### ISSUE THREAD

**Issue by cheney6110** [hackforla/food-oasis#2867]
### Overview
Document does not have a main landmark.One main landmark helps screen reader users navigate a web page.

### Instructions/Resources
https://dequeuniversity.com/rules/axe/4.12/landmark-one-main

### Related issues
- Parent issue: #1355
